src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/services/CamundaService.java

```diff
@@ -75,6 +80,7 @@ private CamundaTask performGetCamundaTaskAction(String id) {
         try {
             return camundaServiceApi.getTask(serviceTokenGenerator.generate(), id);
         } catch (FeignException ex) {
+            log.error("Task Configuration Failure : Error occurred while fetching the task with id '{}'", id);
```

Comment:
```suggestion
            log.error("Error occurred while fetching the task with id '{}'", id);
```
when logging this messages they should be logging under the class name 
CamundaService.java : so I don't know if it's useful to add the Task Configuration Failure , also some of this methods are re-used so could be miss leading, as an example this method performGetCamundaTaskAction() will also be called when retrieving a task so that could be confusing.

Reply:
The text will be used for alerts. I see its called in multiple steps but the are part of Task Configuration

Reply:
Not all of them are part of task configuration for in this example when you call GET /task/{id} this bit of code will be called and if you add this bit the and it fails then you might get this exception when retrieving a task by Id